src/main/scala/com.snowplowanalytics/schemaguru/utils/FileUtils.scala

```diff
@@ -32,19 +67,27 @@ object FileUtils {
    * @param content Content of file
    * @return a success or failure string about the process
    */
-  def writeToFile(fileName: String, fileDir: String, content: String): Validation[String, String] = {
+  def writeToFile(fileName: String, fileDir: String, content: String, force: Boolean = false): Validation[String, String] = {
     val path = fileDir + "/" + fileName
     try {
       makeDir(fileDir) match {
         case true => {
           // Attempt to open the file...
           val file = new File(path)
-
-          // Print the contents of the list to the new file...
-          printToFile(file) { _.println(content) }
-
-          // Output a success message
-          s"File [${path}] was written successfully!".success
+          lazy val contentChanged = isNewContent(file, content)
+          if (!file.exists()) {
+            // Print the contents of the list to the new file...
+            printToFile(file)(_.println(content))
+            // Output a success message
+            s"File [${file.getAbsolutePath}] was written successfully!".success
+          } else if (file.exists() && contentChanged && !force) {
+            s"File [${file.getAbsolutePath}] already exists and probably was modified manually. You can use --force to override".failure
+          } else if (contentChanged && force) {
+            printToFile(file)(_.println(content))
+            s"File [${file.getAbsolutePath}] was overriden successfully!".success
+          } else {
+            s"File [${file.getAbsolutePath}] was not modified".success
```

Comment:
Open question: in the case where the file already exists and the content hasn't changed and the force flag is set, should we be modifying the date anyway to mark that the file has been updated recently and to make the behaviour of the force flag consistent (so that it always overwrites the file)?